### PR TITLE
[bn] localization content/bn/docs/setup/learning-environment/_index.md added

### PR DESCRIPTION
**Pull request by sajibAdhi**
Signed-off-by: Sajib Adhikary <tosajibadhi@gmail.com>
Localizes ``content/bn/docs/setup/learning-environment/_index.md``  to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

**Comment by mitul3737** [comment]
@sajibAdhi Kindly localize the file. You have just changed the title but nothing else.

**Comment by sajibAdhi** [comment]
> @sajibAdhi Kindly localize the file. You have just changed the title but nothing else.

@mitul3737 Kindly check the update of the file.

File: content/bn/docs/setup/learning-environment/_index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<!--
+## কাইন্ড (kind)
+
+[`kind`](https://kind.sigs.k8s.io/docs/) আপনাকে আপনার স্থানীয় কম্পিউটারে Kubernetes চালাতে দেয়। এই টুলের জন্য আপনার [ডকার (docker)](https://docs.docker.com/get-docker/) ইনস্টল এবং কনফিগার করা প্রয়োজন।
+
+কাইন্ড (kind) [দ্রুত শুরু (Quick Start)](https://kind.sigs.k8s.io/docs/user/quick-start/) পৃষ্ঠাটি আপনাকে দেখায় যে আপনি কাইন্ড (kind) সাথে উঠতে এবং দৌড়াতে আপনাকে কী করতে হবে।
+
+## মিনিকুব (minikube)
+
+`kind` এরমত [`minikube`](https://minikube.sigs.k8s.io/) একটি টুল যেটি আপনাকে স্থানীয়ভাবে কুবারনেট (Kubernetes) চালাতে দেয়। `minikube` আপনার ব্যক্তিগত কম্পিউটারে (Windows, macOS এবং Linux PC সহ) একটি একক-নোড (single-node) কুবারনেট (Kubernetes) ক্লাস্টার (cluster) চালায় যাতে আপনি কুবারনেট (Kubernetes) ব্যবহার করে দেখতে পারেন, অথবা দৈনন্দিন উন্নয়ন (development) কাজের জন্য।
```
Comment (mitul3737) [inline content/bn/docs/setup/learning-environment/_index.md:23]:
```suggestion
`kind` এরমত [`minikube`](https://minikube.sigs.k8s.io/) একটি টুল যেটি আপনাকে স্থানীয়ভাবে কুবারনেটিস (Kubernetes) চালাতে দেয়। `minikube` আপনার ব্যক্তিগত কম্পিউটারে (Windows, macOS এবং Linux PC সহ) একটি একক-নোড (single-node) কুবারনেটিস (Kubernetes) ক্লাস্টার (cluster) চালায় যাতে আপনি কুবারনেটিস (Kubernetes) দৈনন্দিন উন্নয়ন (development) কাজের জন্য ।
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**